plugin.video.tyt/main.py

```diff
@@ -0,0 +1,187 @@
+import os, sys, xbmcgui, xbmcplugin, xbmcaddon, json
+import httplib
+
+from resources.lib.modules import logon
+from resources.lib.modules import type1
+from urlparse import parse_qsl
+
+PLUGIN_ID = 'plugin.video.tyt'
+_handle = int(sys.argv[1])
+_url = sys.argv[0]
+xbmcplugin.setContent(_handle, 'tvshows')
+addon_folder = os.path.join(xbmc.translatePath( "special://profile/addon_data/" ), PLUGIN_ID)
+
+settings = xbmcaddon.Addon(id=PLUGIN_ID)
+user_name = settings.getSetting("username")
+user_pwd = settings.getSetting("password")
+COOKIE_FILE = 'special://home/addons/{0}/resources/cookie.txt'.format(PLUGIN_ID)
+cookie = {}
```

Comment:
Please remove this it's not used and is confusing as it looks like you will write into the add-on directory brother than user_data